.github/workflows/zitadel-pr.yml

```diff
@@ -18,28 +18,19 @@ jobs:
         with:
           driver: docker
           install: true
+      - name: Install GoReleaser
+        uses: goreleaser/goreleaser-action@v3
+        with:
+          install-only: true
+          version: v1.10.3
```

Comment:
Please reuse the existing version of go releaser ` version: v1.8.3`

Reply:
done